mypy/checkmember.py

```diff
@@ -180,6 +183,7 @@ def analyze_member_var_access(name: str, itype: Instance, info: TypeInfo,
     This is logically part of analyze_member_access and the arguments are
     similar.
     """
```

Comment:
Describe `report_type`. It looks like there's a new use for it -- the name doesn't look very descriptive any more.